### PR TITLE
@W-13893187 feat : added shipping entries in store setup. 

### PR DESCRIPTION
**Pull request by imAdityaSrivastav**
### What does this PR do?

Adds initialise shipping method to the setup script that populates the inital shipping entities for Native Shipping. 

### What issues does this PR fix or reference?
[W-13893187](https://gus.lightning.force.com/lightning/r/a07EE00001XiU4FYAV/view)

### How to Test/Testing Effort 

- Create and setup the store using commerce:store:create 
- Check for values under developer console
- Entities added : 
      ShippingConfigurationSet
      ShippingRateArea
      ShippingRateGroup
      StandardShippingRate

### CODE REVIEW

File: src/commands/commerce/store/quickstart/setup.ts [hunk 1]
```diff
@@ -488,6 +495,144 @@ export class StoreQuickstartSetup extends SfdxCommand {
...
+            this.flags,
+            this.logger
+        ).result.records[0].Id;
+
+        this.ux.log(`Created ShippingRateArea with id : '${shippingZoneId}'`);
+
+        // Creating StandardShippingArea
+        const isMultiCurrencyEnabled = currencySettings['records'][0]['IsMultiCurrencyEnabled'];
+
+        if (isMultiCurrencyEnabled) {
```
Comment (tarcang) [inline src/commands/commerce/store/quickstart/setup.ts:572]: Can we add tests for these two code paths?

**Review by tarcang — CHANGES_REQUESTED** [review]

File: src/commands/commerce/store/quickstart/setup.ts (hunk 1 above)
Comment (imAdityaSrivastav) [reply src/commands/commerce/store/quickstart/setup.ts:572]: Just to confirm [this](https://github.com/forcedotcom/commerce-on-lightning/blob/develop/test/commands/commerce/store/quickstart/setup.test.ts) is where test should be added right ?

Comment (tarcang) [reply src/commands/commerce/store/quickstart/setup.ts:572]: Yes 👍

Comment (imAdityaSrivastav) [reply src/commands/commerce/store/quickstart/setup.ts:572]: Done.

File: test/commands/commerce/store/quickstart/setup.test.ts [hunk 2]
```diff
@@ -91,6 +92,95 @@ const testFile = `<?xml version="1.0" encoding="UTF-8"?>
 </Network>
 `;
 
+describe('commerce:store:qickstart:setup', () => {
```
Comment (tarcang) [inline test/commands/commerce/store/quickstart/setup.test.ts:95]: nit: typo

File: test/commands/commerce/store/quickstart/setup.test.ts [hunk 3]
```diff
@@ -91,6 +92,95 @@ const testFile = `<?xml version="1.0" encoding="UTF-8"?>
...
+    });
+
+    afterEach(() => {
+        loggerStub.restore();
+        forceDataSoqlStub.restore();
+        forceDataRecordCreateStub.restore();
+        getApplicationContextFromWebStoreIdStub.restore();
+    });
+
+    it('shouldPopulateCurrencyIsoCodeIfOrgIsMulticurrency', async () => {
```
Comment (tarcang) [inline test/commands/commerce/store/quickstart/setup.test.ts:148]: You can write in a human readable format here, no need for camelCase. Spaces are fine.

**Review by tarcang — APPROVED** [review]
Looks good! You can address the comments if you want it but I think my comments can be also addressed later.

**Comment by imAdityaSrivastav** [comment]
@KalyanBoya @wei-liu-salesforce review please. 

Thanks.

**Review by tarcang — APPROVED** [review]

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@salesforce/commerce",
-    "version": "248.1.5",
```
Comment (tarcang) [inline package.json]: Normally this version changes after every commit, but if you don't want to touch the version, I am fine with it. I can update this when I release it.

**Review by tarcang — APPROVED** [review]
LGTM

**Comment by imAdityaSrivastav** [comment]
@tarcang @joeluong-sfcc please merge.

**Pull request merged**